client/js/lounge.js

```diff
@@ -487,11 +487,13 @@ $(function() {
 	}).find("input")
 		.trigger("change");
 
-	$("#badge").on("change", function() {
+	$("#desktopNotifications").on("change", function() {
 		var self = $(this);
 		if (self.prop("checked")) {
 			if (Notification.permission !== "granted") {
-				Notification.requestPermission();
+				Notification.requestPermission(function(){
+					updateDesktopNotificationStatus();
```

Comment:
You can write this as `Notification.requestPermission(updateDesktopNotificationStatus);`